### PR TITLE
Displayed paymentmethods in frontend are ordered correctly

### PR DESCRIPTION
**Pull request by krtschmr**
https://github.com/spree/spree/issues/7515

### CODE REVIEW

File: core/app/models/spree/payment_method.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class PaymentMethod < Spree::Base
 
     scope :active,                 -> { where(active: true) }
     scope :available,              -> { active.where(display_on: [:front_end, :back_end, :both]) }
-    scope :available_on_front_end, -> { active.where(display_on: [:front_end, :both]) }
+    scope :available_on_front_end, -> { active.where(display_on: [:front_end, :both]).order(position: :asc) }
```
Comment (vishalzambre) [inline core/app/models/spree/payment_method.rb]: @krtschmr  Its good if we append ```order(position: :asc)```  for active scope, which internally used in rest of scopes.
 @damianlegawiec What's your thoughts here?

Comment (damianlegawiec) [inline core/app/models/spree/payment_method.rb]: @vishalzambre good idea!

Comment (krtschmr) [inline core/app/models/spree/payment_method.rb]: yes i agree

to order the available scope makes more sense though.

**Comment by giubueno** [comment]
How about specifying which field is being used to order the payment?

**Comment by krtschmr** [comment]
@giubueno  i don't understand what you mean with *field*.  A Payment method has the field *position*

**Comment by giubueno** [comment]
I didn't see the position attribute :)

**Comment by krtschmr** [comment]
closed in favor of https://github.com/spree/spree/pull/7538

**Pull request closed without merging**